modules/base/tool.go

```diff
@@ -143,9 +144,9 @@ func FileSize(s int64) string {
 }
 
 // PrettyNumber produces a string form of the given number in base 10 with
-// commas after every three orders of magnitud
-func PrettyNumber(v int64) string {
-	return humanize.Comma(v)
+// commas after every three orders of magnitude
+func PrettyNumber(i interface{}) string {
+	return humanize.Comma(util.NumberIntoInt64(i))
```

Comment:
How about a very big number like 30000, whether it will display `30K`?

Reply:
Currently this function just does formatting and not rewriting it to `K`, `M`. I'm not sure if that needs to be locale-aware but seems nice to be improved and take that into account.

Reply:
if you do, lower-case k please. K == Kelvin as units are capital letters, why k=kilo is a multiplier